internal/static/embed/cleanup-instance.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xeu
```

Comment:
Tiny nit but I'm not seeing any bashisms in there, so you could use `#!/bin/sh` instead of `#!/bin/bash` to save a few ms of time :)